app/controllers/ContactHmrcController.scala

```diff
@@ -72,17 +72,17 @@ class ContactHmrcController @Inject() (
   contactHmrcPage: ContactHmrcPage,
   contactHmrcConfirmationPage: ContactHmrcConfirmationPage,
   headerRetriever: RefererHeaderRetriever
-)(implicit val appConfig: AppConfig, val executionContext: ExecutionContext)
+)(using val appConfig: AppConfig, val executionContext: ExecutionContext)
     extends FrontendController(mcc)
     with DeskproSubmission
     with I18nSupport {
 
-  implicit def lang(implicit request: Request[_]): Lang = request.lang
+  given lang(using request: Request[?]): Lang = request.lang
 
-  def index(service: Option[String], userAction: Option[String], referrerUrl: Option[ReferrerUrl]) =
-    Action.async { implicit request =>
+  def index(service: Option[String], userAction: Option[String], referrerUrl: Option[ReferrerUrl]): Action[AnyContent] =
+    Action.asyncUsing { request ?=>
```

Comment:
example with the extension method - but could just have an explicit `given ...` instead

Reply:
removed in favour of explicit `given`s